environment_unix.yml

```diff
@@ -45,4 +45,5 @@ dependencies:
   - rapidcheck
   - remotery
   - robin_hood
-  - lmdbxx
\ No newline at end of file
+  - lmdbxx
+  - arrow-cpp
```

Comment:
@jjerphan  Do you know what is the difference between libarrow and arrow-cpp in conda?

Reply:
From [Apache Arrow feedstock](https://github.com/conda-forge/arrow-cpp-feedstock/blob/main/recipe/meta.yaml#L185-L186), `arrow-cpp` was used before Apache Arrow 10.0.0. `arrow-cpp` seems now to be an alias for `libarrow` to maintain retro-compatibility: reading [`arrow-cpp`'s constraints](https://github.com/conda-forge/arrow-cpp-feedstock/blob/main/recipe/meta.yaml#L188-L196), this retrocompatibility is made possible by requiring `libarrow` for `arrow-cpp` at all stages.

Hence, I think we should specify `libarrow` over `arrow-cpp`.

Reply:
Actually, [`arrow-cpp` is also required for `libarrow`](https://github.com/conda-forge/arrow-cpp-feedstock/blob/main/recipe/meta.yaml#L136-L137) so installing one of them will install both.

I still think that using `libarrow` is semantically preferable, but that's a nitpick/personal preference.